examples/atomic-registry/systemd/install.sh

```diff
@@ -1,66 +1,109 @@
-#!/bin/bash
+#!/usr/bin/env bash
+
+set -o pipefail
```

Comment:
Not sure if they're set elsewhere, but in general we want our Bash scripts to run with:
```
set -o errexit
set -o nounset
set -o pipefail
```

An easy way to get this is to `source hack/lib/init.sh`, but I'm not sure if that's appropriate for this script.